### DISCUSSION TITLE
Use of "variation" tab in Area Settings

### DISCUSSION THREAD

**Discussion by manolof** [theyosh/TerrariumPI#894] [Q&A] [answered]
Hello there!

I was wondering if someone could help me with the Variation tab inside Area Settings.
I have a case scenario, but can't figure out the best setting for this.

For my enclosure I would like the temperate alarm to be set lower during the "winter" season, and back to normal during the warm season.
For this, I would like to use either a dateTime, or a JSON file with a list of dateTimes.
Example:
![image](https://github.com/theyosh/TerrariumPI/assets/3025620/47c38be1-22f2-49e7-a6c1-751e458df39c)

With the current documentation, it is not clear what the format for the "At time" should be, or for the other options either.

Any help much appreciated!

**Comment by HTPProXy** [comment, marked as answer]
Hello manolof,
it may be a little late but just found your discussion. As far as I know the variation tab is used to define different values throughout a day. So you can have a target temperature at a given time and another at another time. As far as I know it is not possible to use the variation tab to define different seasons.
